CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# v1.13.2
+
+## Release v1.13.2 - 2021/07/12
+
+### Enhancement
+
+* fluent-plugin-generate: Storage plugin was supported.
+  https://github.com/fluent/fluentd/pull/3426
+  https://github.com/fluent/fluentd/pull/3444
```

Comment:
#3444 isn't a feature enhancement, it's a bug fix, isn't it?

Reply:
Yes, but it was not released at that time, so I listed here.

Reply:
https://github.com/fluent/fluentd/issues/3442 was reported against master version.

Reply:
@ashie  should #3444 be omitted not to confuse?

Reply:
The bug #3442 exists since v1.12.3

https://github.com/fluent/fluentd/commit/b6e76c09d60ee866de6470f2bdd2551e7c8591d3

Reply:
And I confirmed the bug by released 1.13.1.

Reply:
Ah, I've misunderstood.
I'll fix.

Reply:
Fixed.